### ISSUE TITLE
Feature Request: add blacklist option to preferences

### ISSUE THREAD

**Issue by dgplayer** [RequestPolicy/requestpolicy#438]
Any destination on the blacklist should be silently blocked on all web pages without the Request Policy icon showing red and without being listed in the drop-down menu

**Comment by HaraldKi** [comment]
I would like to second this. If I explicitly blacklist a site, the flag does not need to go red if that site is blocked.

**Comment by ghost** [comment]
I am waiting blacklist option. It it will be great. Thank you!

**Comment by myrdd** [comment]
yes, I'm planning this. blacklists aren't implemented yet, but I've changed the color behavior of the flag [on my fork](https://github.com/RequestPolicyContinued/requestpolicy) – it gets red _only_ if there is at least one request which has been blocked by the "block by default" policy. Any feedback is very welcome!

**Comment by ghost** [comment]
I too would like to see this, as this would complete the add-on in my opinion.

**Comment by moshpirit** [comment]
@myrdd I'm using Policy Request Continued and I can't find how to use the blacklist.

**Comment by dgplayer** [comment]
First upgrade to the latest version of "Request Policy Continued"
Then either click on the tool bar icon [red or grey flag]
or the RequestPolicy Continued submenu on the Tools menu.

To allow a connection:
Click on a blocked destination on the left side of the dialog, then
click on an allow option on the right side.
     allow from domain to domain
     temp allow from domain to domain
     allow from \* to domain
     temp allow from \* to domain

To block a connection [use for ad servers and tracking analytics]:
Click on an allowed destination on the left side of the dialog, then
click on an allow option on the right side.
     block from \* to destination domain
     temp block from \* to destination domain

If you cannot decide if a site is good or bad, then google the site;
If it is an infrastructure site [like jwpsrv.com or fonts.googleapis.com] allow it
If it is a tracking, ad server or analytics site [like gooletagservices.com or google-analytics.com] block it;
If you cannot determine what a site does, block it.

**Comment by moshpirit** [comment]
I've installed the beta and now I can see it, thank you! 
BTW, congrats for the welcome tutorial, it's pretty nice, you're a great developer: very good program and a cool presentation :+1:

**Comment by dgplayer** [comment]
Remember, blocking a destination may break a web page.
If so then, temp allow blocked destinations and reload the web page
until the web page functions again.

Permanently allow the last unblocked destination or
if it is doing something that you will not tolerate leave it blocked
and temporarily allow it only when you need to.

Sometimes it is better to leave a site partly broken in order to preserve your privacy.

TIP: if you are on a web page with embedded videos and the videos are not displayed,
right click on the blank box where the video would be displayed and select the context
menu: "Inspect Element"; generally a link to the video will within a few lines above or below
the line selected in the Inspector. Right Click on the HTML element with the link and select
"Edit as HTML"; you can then cut and paste the link into a new browser window and view
the video.

**Comment by moshpirit** [comment]
Thank you! I've been using it for a while and I use NoScript as well, so I know that if the website get broken, I shouldn't have a heart attack :) I appreciate the recommendations anyway. For YouTube videos I use TOR browser and HTML5 videos get loaded perfectly on Firefox :) (and I don't use Flash)
